### ISSUE TITLE
Floating spaces

### ISSUE THREAD

**Issue by bartzy** [aeagle/react-spaces#19]
Thanks for publishing this project!

It could be nice to have an option to configure some spaces as floating above their siblings, so they don't take any space from them.
Could be an easy way to implement something like a left/right drawer menu.

**Comment by aeagle** [comment]
@bartzy I've had a thought about this and the best way to implement this. The most obvious way would be to introduce a floated property on an anchored space.

However a more flexible way I believe would be to introduce a layering concept. Therefore within one space you could introduce layers in which spaces would affect the spaces in the same layer (as they do now) but not spaces in different layers. Something like:

~~~~
<ViewPort>

  <Layer zindex={10}>

    <Left size={xxx}>
       ... floating sidebar
    </Left>

  <Layer>

  <Fill>
      ... this takes up the full page 
  </Fill>

</ViewPort>
~~~~

Despite being in different layers, the Fill and the Left are still constrained by their common parent ViewPort but being in different layers do not affect each other.

**Comment by thekapadia** [comment]
@aeagle - when do you plan to release this? I'm eagerly waiting to use this feature in my project.

**Comment by aeagle** [comment]
I've just released version 0.1.15 with the layers functionality. Here is an example of usage https://codesandbox.io/s/react-spaces-layers-dsj19?fontsize=14

**Comment by aeagle** [comment]
Hi @Slapbox 

Thanks for the suggestion. I'll sort out a redirect shortly for the old demos to the storybook versions.

In terms of your other requests it would be good to setup an feature request for each of the following:

- Optional corner resize handles - something I thought about adding myself already 
- Positioned space combined left/right or top/bottom resizing - it would be interesting to know the use case for this but we can continue the discussion on a separate issue

Could you add these with details and I can look to getting this functionality added?
